generatorv2/test/utils_test.rb

```diff
@@ -22,15 +22,15 @@ def test_underscore_with_two_words
       Generator.new("two-fer").underscore("two-fer")
   end
 
-  def test_status
+  def test_skip
```

Comment:
Should the name be `skip_test` as that is what it does, rather than it testing a skip?